File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/app/IQChannelsListener.java
/*
 * Copyright (c) 2017 iqstore.ru.
 * All rights reserved.
 */

package ru.iqchannels.sdk.app;

import ru.iqchannels.sdk.schema.ClientAuth;

/**
 * Created by Ivan Korobkov i.korobkov@iqstore.ru on 14/08/2017.
 */

public interface IQChannelsListener {
    void authenticating();
    void authComplete(ClientAuth auth);
    void authFailed(Exception e);
}

File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/app/IQChannelsListener.kt
/*
 * Copyright (c) 2017 iqstore.ru.
 * All rights reserved.
 */
package ru.iqchannels.sdk.app

import ru.iqchannels.sdk.schema.ClientAuth

interface IQChannelsListener {
	fun authenticating()
	fun authComplete(auth: ClientAuth)
	fun authFailed(e: Exception, attempt: Int)
}
